src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -122,18 +122,36 @@ GHRelease wrap(GHRepository owner) {
      * http://stackoverflow.com/questions/12361090/server-name-indication-sni-on-java but involve more complicated
      * handling of the HTTP requests to github's API.
      *
+     * @param file file to upload
+     * @param contentType content type
+     * @param uploadUrlPrefix URL prefix for uploading; for example, for github.com it's https://uploads.github.com
+     * @return instance of GHAsset uploaded
+     * @since 1.75
      * @throws IOException
      */
-    public GHAsset uploadAsset(File file, String contentType) throws IOException {
+    public GHAsset uploadAsset(File file, String contentType, String uploadUrlPrefix) throws IOException {
         Requester builder = new Requester(owner.root);
 
-        String url = format("https://uploads.github.com%s/releases/%d/assets?name=%s",
-                owner.getApiTailUrl(""), getId(), file.getName());
+        String url = format("%s%s/releases/%d/assets?name=%s",
+                            uploadUrlPrefix, owner.getApiTailUrl(""), getId(), file.getName());
```

Comment:
can u move default prefix to const and make last arg optional (in case of null arg, use default)?

`defaultIfNull(uploadUrlPrefix, DEFAULT_UPLOAD_URL)`
It can be useful instead of manually define in each place both enterprise and not enterprise urls (for example in github-jenkins-plugin)